scout/commands/update/panel.py

```diff
@@ -46,10 +60,46 @@ def panel(panel, version, update_date, update_version):
             LOG.warning(err)
             click.Abort()
 
+    # Any mintainer updates?
+
+    new_maintainer = None
+    if add_maintainer:
+        if add_maintainer not in [user["_id"] for user in adapter.users()]:
```

Comment:
```suggestion
        user_obj = adapter.user(user_id=add_maintainer)
        if not user_obj:
```
To avoid looping over all users

Reply:
Hm, I considered that, but needed it for some reason. Perhaps that was earlier, or I was just imagining things. Will check!

Reply:
No, can't remember why; probably some older idea. Thanks!